spec/controllers/socure_webhook_controller_spec.rb

```diff
@@ -4,6 +4,17 @@
 
 RSpec.describe SocureWebhookController do
   describe 'POST /api/webhooks/socure/event' do
+    let(:user) { create(:user) }
+    let(:socure_docv_transaction_token) { 'dummy_docv_transaction_token' }
+    let(:document_capture_session) do
+      DocumentCaptureSession.create(
+        user_id: user.id,
+        user: user,
+        requested_at: Time.zone.now,
+        socure_docv_transaction_token: socure_docv_transaction_token,
+      )
```

Comment:
I found this in the code elsewhere, so I started using it. Let's stick with it just for consistency:

```suggestion
      DocumentCaptureSession.create(user:).tap do |dcs|
        dcs.socure_docv_transaction_token = socure_docv_transaction_token
      end
```